scrapers/kodi.yml

```diff
@@ -0,0 +1,10 @@
+name: "Kodi XML"
+sceneByFragment:
+    action: script
+    script:
+      - python
+      # use python3 instead if needed
```

Comment:
According to documentation, Stash is supposed to detect which python is available on the system : either python or python3. See: https://docs.stashapp.cc/in-app-manual/scraping/scraperdevelopment/#actions

If the documentation affirmation stands, this comment may be deleted.